api/server/router/image/image_routes.go

```diff
@@ -141,7 +141,7 @@ func (ir *imageRouter) postImagesCreate(ctx context.Context, w http.ResponseWrit
 		id, progressErr = ir.backend.ImportImage(ctx, tagRef, platform, comment, layerReader, r.Form["changes"])
 
 		if progressErr == nil {
-			output.Write(streamformatter.FormatStatus("", id.String()))
+			_, _ = output.Write(streamformatter.FormatStatus("", id.String())) //nolint:govet // ignore printf: non-constant format string in call to github.com/docker/docker/pkg/streamformatter.FormatStatus (govet)
```

Comment:
Fix it; don't ignore it.
```suggestion
			_, _ = output.Write(streamformatter.FormatStatus("", "%v",  id.String()))
```